### PR TITLE
Slightly reworks abductee objectives

### PR DESCRIPTION
**Pull request by LodedDiper69**
## About The Pull Request
Adds several new objectives, three of those with custom variables, and removes two old ones.

## Why It's Good For The Game
People often ignore ayy objectives because they're unfun. This PR will hopefully remove the worst ones and replace them with objectives that are slightly more interesting.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: Added 11 new abductee objectives.
del: Removed 2 old abductee objectives.
/:cl:

### CODE REVIEW

File: strings/abductee_objectives.txt [hunk 1]
```diff
@@ -38,7 +37,14 @@ You see you see what they cannot you see the open door you seeE you SEeEe you SE
...
-You are in a action movie. You must say as many cheesy one liners as possible.
 You are a military instructor. You must make sure the crew is in top shape for the war against the syndicate!
 You are a writer. Convince the crew of your superb writing skills!
 You know something bad is about to happen to this station. Convince the crew to get off of it while they still can!
+You are a landlord. Barricade dormitories and demand rent from anyone trying to enter.
+You are a cyborg. Follow AI's orders, state laws when needed, but don't get your brain removed.
+Goblins are real. Gather evidence to prove their existence and show it to the world!
+Start a pyramid scheme with you at the top.
+Abductors are just a silly conspiracy theory. Debunk the crew's hysteria with your brainpower.
+You are a euphoric Atheist. Argue with chaplain over the existence of any gods, and don't forget your fedora.
```
Comment (itsmeow) [inline strings/abductee_objectives.txt:48]: euphoric? do you mean enthusiastic?

File: strings/abductee_objectives.txt [hunk 2]
```diff
@@ -38,7 +37,14 @@ You see you see what they cannot you see the open door you seeE you SEeEe you SE
...
+You are feeling a calling from a long-gone nation. Inject everyone with the chav mutator and hand out knifes and baked beans.
```
Comment (itsmeow) [inline strings/abductee_objectives.txt:49]: knives

File: strings/abductee_objectives.txt [hunk 3]
```diff
@@ -38,7 +37,14 @@ You see you see what they cannot you see the open door you seeE you SEeEe you SE
...
+There is no such thing as an unrevivable corpse. Insert a monkey brain into them and get these lazy people back to their departments!
```
Comment (itsmeow) [inline strings/abductee_objectives.txt:50]: maybe department lock this to medical?

File: strings/abductee_objectives.txt [hunk 4]
```diff
@@ -29,7 +29,6 @@ The crew must get to know one another better. Break down the walls inside the st
 The CEO of Nanotrasen is coming! Ensure the station is in absolutely pristine condition.
 Flood the station's powernet with as much electricity as you can.
 Replace all the floor tiles with wood, carpeting, grass or bling.
-You must escape the station! Get the shuttle called!
 Don't allow anyone to be cloned.
 The oxygen is killing them all and they don't even know it. Make sure no oxygen is on the station.
 Your body must be improved. Ingest as many drugs as you can.
```
Comment (itsmeow) [inline strings/abductee_objectives.txt:34]: this one doesn't really create any good RP

File: strings/abductee_objectives.txt [hunk 5]
```diff
@@ -29,7 +29,6 @@ The crew must get to know one another better. Break down the walls inside the st
 The CEO of Nanotrasen is coming! Ensure the station is in absolutely pristine condition.
 Flood the station's powernet with as much electricity as you can.
```
Comment (itsmeow) [inline strings/abductee_objectives.txt:30]: could use some flavortext

File: strings/abductee_objectives.txt [hunk 6]
```diff
@@ -29,7 +29,6 @@ The crew must get to know one another better. Break down the walls inside the st
...
 Replace all the floor tiles with wood, carpeting, grass or bling.
```
Comment (itsmeow) [inline strings/abductee_objectives.txt:31]: also could use flavor

File: strings/abductee_objectives.txt [hunk 7]
```diff
@@ -29,7 +29,6 @@ The crew must get to know one another better. Break down the walls inside the st
...
-You must escape the station! Get the shuttle called!
 Don't allow anyone to be cloned.
```
Comment (itsmeow) [inline strings/abductee_objectives.txt:32]: this one seems more annoying than fun

File: code/modules/antagonists/abductor/abductee/abductee_objectives.dm [hunk 8]
```diff
@@ -27,6 +27,27 @@
 	var/style = pick(list("pantomime", "rhyme", "haiku", "extended metaphors", "riddles", "extremely literal terms", "sound effects", "military jargon", "three word sentences"))
 	explanation_text+= " [style]."
 
+/datum/objective/abductee/cargonia
+	explanation_text = "Your comrades have been oppressed for long enough. Declare and protect the independence of"
+
+/datum/objective/abductee/cargonia/New()
+	var/department = pick(list("service", "security", "command", "cargo", "medbay", "science", "engineering"))
```
Comment (Penwin0) [inline code/modules/antagonists/abductor/abductee/abductee_objectives.dm:34]: Should probably get the department they belong to here

**Comment by LodedDiper69** [comment]
gonna have to close it for now.

**Pull request closed without merging**